pkg/tests/integration/test_salt_ufw.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os.path
+import pathlib
+import subprocess
+
+import pytest
+from pytestskipmarkers.utils import platform
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.skip_on_windows()
+def test_salt_ufw(salt_master, install_salt):
+    """
+    Test salt.ufw for Debian/Ubuntu salt-master
+    """
+    log.warning(f"DGM test_salt_ufw install_salt '{install_salt}'")
+
```

Comment:
```suggestion
    log.warning(f"DGM test_salt_ufw install_salt '{install_salt}'")

```
Left over?
If useful, lower the log level, but I think it can go.

Reply:
I should have added WIP, wanted to get the work saved off while addressed a different issue, and built so that I can try the new package testing instructions. So disregard this test as it will change. Btw: use warning log level for debug stuff since then don't have to bother with editing config file for minion